src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -359,7 +359,10 @@ void primitive_inst::update_impl() {
                             return;
                     }
 
-                    auto impl = _node->type()->choose_impl(*_node, updated_params);
+                    auto static_params = updated_params;
+                    static_params.is_dynamic = false;
```

Comment:
If we have this flag in kernel param only, not in the impl param, this is unnecessary.

Reply:
Yeah, thank you